aspnet/fundamentals/logging.rst

```diff
@@ -162,7 +182,7 @@ Examining the Application event log in the Windows Event Viewer, the following e
 Configuring Other Providers
```

Comment:
I think this section needs to be removed. We prototyped support for NLog, Serilog and other logging providers, but we don't actually ship them. Support for those libraries still needs to come from those project owners.